<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Temperatura</class>
 <widget class="QDialog" name="Temperatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>364</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>726</width>
    <height>364</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>876</width>
    <height>490</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Temperatura</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/caliente.png</normaloff>:/recursos/caliente.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Lucida Bright</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Grados Cent√≠grados</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLCDNumber" name="lcdCent">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 156, 140);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialCent">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(241, 230, 10);</string>
        </property>
        <property name="minimum">
         <number>-273</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCent">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Lucida Bright</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Grados Fahrenheit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdFahr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46, 204, 113);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialFahr">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 174, 66);</string>
        </property>
        <property name="minimum">
         <number>-459</number>
        </property>
        <property name="maximum">
         <number>212</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFahr">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>Lucida Bright</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Kelvin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdKelv">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(154, 205, 50);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialKelv">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(255, 81, 0);</string>
        </property>
        <property name="maximum">
         <number>373</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblKelv">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKelv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKelv</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
